datamol/viz/_lasso_highlight.py

```diff
@@ -421,6 +425,10 @@ def lasso_highlight_image(
         line_width: width of drawn lines.
         scale_padding: Padding around the molecule when drawing to scale.
         verbose: Whether to print the verbose information.
+        highlight_atoms: The atoms to highlight, a list for each molecule.
```

Comment:
What's the difference between `atom_indices` and `highlight_atoms`?

Reply:
- `atom_indices` is for making the lasso highlight around a set of atoms. 
- `highlight_atoms` is for the drawer object itself and will be used for example to change the colors of atoms. It does not impact the lasso process. 

I have updated the doc to reflect that. 

Thanks